From cc8e6930a486495bd948b564b543f982b7f4585e Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Sat, 6 May 2017 17:50:17 +0800
Subject: [PATCH] arm64: rockchip_defconfig: rename UID_CPUTIME to
 UID_SYS_STATS

2ea16502ca00 ("ANDROID: uid_sys_stats: rename uid_cputime.c to uid_sys_stats.c")

Change-Id: I11ad23129722f87111a5c6f3881b4bdcc5626f66
Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 9328221a997d..ed0119fafe16 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -235,7 +235,7 @@ CONFIG_BLK_DEV_RAM_SIZE=8192
 CONFIG_BLK_DEV_NVME=y
 CONFIG_ROCKCHIP_SCR=y
 CONFIG_SRAM=y
-CONFIG_UID_CPUTIME=y
+CONFIG_UID_SYS_STATS=y
 CONFIG_USB_CAM_GPIO=y
 CONFIG_SCSI=y
 CONFIG_BLK_DEV_SD=y
-- 
2.35.3

